src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -201,10 +174,24 @@ describe('Step Users', () => {
       await goToRegistrationStep();
       await clickRegisterLater();
       await goToUsersStep();
-      await addInvalidUser();
+      await clickAddUser();
+      await addUserName('ss.');
+      await addSshKey('ssh');
       const invalidUserMessage = screen.getByText(/invalid user name/i);
       await waitFor(() => expect(invalidUserMessage));
     });
+
```

Comment:
'with invalid name' and 'with invalid ssh key' should be moved to the 'Step Users' describe as they're not testing generated blueprint request.